lib/mix/phoenix/schema.ex

```diff
@@ -343,4 +346,19 @@ defmodule Mix.Phoenix.Schema do
     |> String.trim_leading("_")
     |> String.replace("/", "_")
   end
+
+@doc """
```

Comment:
We shouldn't include docs for private functions.

Also, can please align the definition below with [the one in Ecto.SQL here](https://github.com/elixir-ecto/ecto_sql/blob/d3fc4a8cc03ef601c7158d3cbdb2b477e22e7719/lib/mix/tasks/ecto.gen.migration.ex#L103-L108)?

Reply:
Corrected, thank you.